keps/sig-apps/3998-job-success-completion-policy/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: Job success/completion policy
+kep-number: 3998
+authors:
+  - "@tenzen-y"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+status: implementable
+creation-date: 2023-06-05
+reviewers:
+  - "@mimowo"
+  - "@alculquicondor"
```

Comment:
@kannon92 and @vsoch deserve to be in this list too

Reply:
Done.